lib/couchx/prepare_query.ex

```diff
@@ -76,10 +76,11 @@ defmodule Couchx.PrepareQuery do
     %{"$or": [Enum.reduce(list, %{}, &Map.merge(&2, build_field_condition(&1)))] }
   end
 
-  defp parse_options(%{order_bys: order_bys, limit: limit}) do
+  defp parse_options(%{order_bys: order_bys, limit: limit, offset: skip}) do
```

Comment:
if offset is not always present this will be skiped

Reply:
if there is no offset, it is nil, what parse_options receive is #Ecto.Query<>

Reply:
As it receive an Ecto query , parse_options always have these keys

```elixir
iex()> Struct |> Ecto.Queryable.to_query() |> Map.keys()
[:__struct__, :aliases, :assocs, :combinations, :distinct, :from, :group_bys,
 :havings, :joins, :limit, :lock, :offset, :order_bys, :prefix, :preloads,
 :select, :sources, :updates, :wheres, :windows, :with_ctes]
```
I validate that it works even without query options and works fine
```elixir
Struct |> Repo.all()
Struct |> Ecto.Queryable.to_query() |> Repo.all()
Struct |> Ecto.Queryable.to_query() |> where(^filters) |> Repo.all()
```